pkg/sdk/metrics.go

```diff
@@ -34,7 +36,15 @@ func ExposeMetricsPort() {
 		logrus.Errorf("failed to initialize service object for operator metrics: %v", err)
 		return
 	}
-	err = Create(service)
+	kubeconfig, err := k8sutil.GetNewKubeConfig()
```

Comment:
Change to `config.GetConfig()` using the controller-runtime helper.
We don't need our own k8sutil for this anymore.